python/tvm/autotvm/tuner/graph_tuning.py

```diff
@@ -0,0 +1,106 @@
+"""Utilities for tuning a whole graph (a set of tasks)"""
+
+import os
+from . import callback, XGBTuner, GATuner, RandomTuner, GridSearchTuner
+from .. import measure, record, task
+
+def tune_tasks(tasks,
+               rpc_device_key,
+
+               tuner='ga',
+               n_trial=500,
+               early_stopping=200,
+               log_filename='tuning.log',
+
+               mea_number=5,
+               mea_parallel_num=1,
+               mea_timeout=20,
+               mea_use_ndk=False,
```

Comment:
is it possible to pass in MeasureOption here? The options seem to be a bit duplicating with MeasureOption